src/screens/SignUp/index.js

```diff
@@ -20,6 +29,79 @@ const Signup = () => {
     e.preventDefault();
   };
 
+  const togglePasswordVisibility = () => {
+    setUser({ ...user, isPasswordShown: !isPasswordShown });
+  };
+
+  const togglePasswordConfirmVisibility = () => {
+    setUser({ ...user, isPasswordConfirmShown: !isPasswordConfirmShown });
+  };
+
+  const passwordToggleEye = {
+    toShowPassword: (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 24 24"
+        fill="black"
+        width="18px"
+        height="18px"
+        className="password-toggle"
+        onClick={togglePasswordVisibility}
+      >
+        <path d="M0 0h24v24H0z" fill="none" />
+        <path d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z" />
+      </svg>
+    ),
+    toHidePassword: (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 24 24"
+        fill="black"
```

Comment:
Let's change the fill on the icons. The black demands a bit too much attention from the eye first thing. Make it a lighter shade of gray ideally.

Reply:
Changed it from black to gray. What do you think?

Reply:
I went in and added a fill on the class on the svg. gray is good.